compio-runtime/src/attacher.rs

```diff
@@ -143,6 +92,20 @@ impl<S> IntoInner for Attacher<S> {
     }
 }
 
+impl<S> Deref for Attacher<S> {
+    type Target = S;
+
+    fn deref(&self) -> &Self::Target {
+        &self.source
+    }
+}
+
+impl<S> DerefMut for Attacher<S> {
+    fn deref_mut(&mut self) -> &mut Self::Target {
+        &mut self.source
+    }
+}
+
 /// Represents an attachable resource to driver.
```

Comment:
I think this need to be an unsafe trait, or should be sealed

Reply:
This trait is useless. I'll remove it.